try {
  print(int('x'))
} catch {
  print('v')
}
